_clients/javascript/index.md

```diff
@@ -140,20 +141,22 @@ async function search() {
 search().catch(console.log);
 ```
 
-## Circuit Breaker
+## Circuit breaker
 
-The `memoryCircuitBreaker` parameter in the [Cluster Settings API]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-settings/) gives you the ability to reject large query responses where the size of the response could crash OpenSearch Dashboards. To set the Circuit Breaker setting, use the `POST _cluster/settings` API operation on your active JS OpenSearch cluster.
+The `memoryCircuitBreaker` option can be used to prevent errors caused by a response payload being too large to fit into the heap memory available to the client.
 
-`memoryCircuitBreaker` contains two fields:
+The `memoryCircuitBreaker` object contains two fields:
 
-- `enabled`: A Boolean used to turn the Circuit Breaker on or off. Defaults to `false`.
-- `maxPercentage`: The threshold that determines whether the Circuit Breaker engages. The input range must be between `[0 ,1]`. Any number that exceeds that range will correct to `1.0`.
+- `enabled`: A Boolean used to turn the circuit breaker on or off. Defaults to `false`.
+- `maxPercentage`: The threshold that determines whether the circuit breaker engages. Valid values are [0, 1]. Any value that exceeds that range will correct to `1.0`.
```

Comment:
I don't think `Valid values are [0, 1]` is a correct statement. **maxPercentage** Can be any real number between 0.0 and 1.0, representing percentage from 0 to 100%. For example the user can use `0.5` to set the percentage to 50%.

Reply:
Reworded.